internal/host/validator.go

```diff
@@ -795,3 +795,42 @@ func (v *validator) printSufficientPacketLossRequirementForRole(c *validationCon
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *validator) isDomainNameResolvedCorrectly(c *validationContext) ValidationStatus {
+	var response models.DomainResolutionResponse
+	if err := json.Unmarshal([]byte(c.host.APIDomainNameResolution), &response); err != nil {
+		v.log.WithError(err).Warn("Parse domain resolution")
+		return ValidationError
+	}
+	if len(response.Resolutions) == 0 {
```

Comment:
This `if` is redundant, the loop already won't run (and then the function will fail) if the length of the list is 0